pipelines/batch/Dockerfile

```
@@ -43,7 +43,7 @@ ENV FHIR_DEBEZIUM_CONFIG_PATH="/workspace/utils/dbz_event_to_fhir_config.json"
 
 
 ENTRYPOINT java -jar /usr/src/Main/app.jar  \
-           --openmrsServerUrl=${OPENMRS_URL} --openmrsUserName=${OPENMRS_USERNAME} --openmrsPassword=${OPENMRS_PASSWORD} \
+           --fhirServerUrl=${OPENMRS_URL}/ws/fhir2/R4 --fhirServerUserName=${OPENMRS_USERNAME} --fhirServerPassword=${OPENMRS_PASSWORD} \
```

This should not require `/ws/fhir2/R4` because that will cause issues for HAPI. Please drop it and also change `OPENMRS` in these env. variables to something more generic, e.g., `FHIR` or `FHIR_SERVER`:
```suggestion
           --fhirServerUrl=${FHIR_SERVER_URL}/ws/fhir2/R4 --fhirServerUserName=${FHIR_SERVER_USERNAME} --fhirServerPassword=${FHIR_SERVER_PASSWORD} \
```